components/event_dispatcher.rst

```diff
@@ -390,7 +398,10 @@ Take the following example of a subscriber that subscribes to the
                     ['onKernelResponsePre', 10],
                     ['onKernelResponsePost', -10],
                 ],
+                // when using two arguments, the event and the event name
                 OrderPlacedEvent::NAME => 'onStoreOrder',
+                // when using only one argument, the event (since Symfony 4.3+)
+                OrderPlacedEvent::class => 'onStoreOrder',
```

Comment:
This sounds a bit confusing to me

cc @javiereguiluz

Reply:
@OskarStark I agree, btu I do not know how I can document both in the same code snippet, or should I duplicate this code block?

Reply:
friendly ping @OskarStark as I remembered this PR today ^^

Reply:
friendly ping @OskarStark @javiereguiluz do you have an idea on the way to document both in this code example?
thanks :)